modules/apps/change-tracking/change-tracking-rest-test/src/testIntegration/java/com/liferay/change/tracking/rest/resource/v1_0/test/BaseCTCollectionResourceTestCase.java

```diff
@@ -436,20 +443,39 @@ protected CTCollection testPostCTCollection_addCTCollection(
 
 	@Test
 	public void testGetCtCollectionsByClassPage() throws Exception {
+		Layout layout = LayoutTestUtil.addTypeContentLayout(testGroup);
+
+		long layoutClassNameId = ClassNameLocalServiceUtil.getClassNameId(Layout.class);
+		long classPK = layout.getPlid();
+
 		Page<CTCollection> page =
-			ctCollectionResource.getCtCollectionsByClassPage(null, null);
+			ctCollectionResource.getCtCollectionsByClassPage(
+				(int) layoutClassNameId, (int) classPK);
 
 		long totalCount = page.getTotalCount();
 
 		CTCollection ctCollection1 =
 			testGetCtCollectionsByClassPage_addCTCollection(
 				randomCTCollection());
 
+		try (SafeCloseable safeCloseable =
+				 CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					 ctCollection1.getId())) {
+			LayoutLocalServiceUtil.updateName(layout, ctCollection1.getName(), layout.getDefaultLanguageId());
+		}
+
 		CTCollection ctCollection2 =
 			testGetCtCollectionsByClassPage_addCTCollection(
 				randomCTCollection());
 
-		page = ctCollectionResource.getCtCollectionsByClassPage(null, null);
+		try (SafeCloseable safeCloseable =
+				 CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(
+					 ctCollection2.getId())) {
+			LayoutLocalServiceUtil.updateName(layout, ctCollection2.getName(), layout.getDefaultLanguageId());
+		}
+
+		page = ctCollectionResource.getCtCollectionsByClassPage(
+				(int) layoutClassNameId, (int) classPK);
```

Comment:
We should not make changes to the base test case because this class is generated. You will lose all your custom changes if you run the rest builder again. If you need to make changes to the test case, you'll need to override it in the test class CTCollectionResourceTest.